### ISSUE TITLE
[suggestion] Heroku deploy option

### ISSUE THREAD

**Issue by luphoria** [sm64js/sm64js#360]
Now i'm not sure exactly the power of heroku cloud computing, but I do know that many many people utilise heroku to deploy applications. I'm not completely certain heroku could host sm64js MMO very well, but maybe a deploy for vanilla (and MMO for the people who pay for higher quality cloud servers)? I think it would be a nice addition that really would open up this project to about anyone (you can deploy servers and updates without ever even needing to leave your browser)

Now i've never really used heroku for much anything, but from my experience it is quite intuitive to set up -- simply add some commands for heroku to run on startup (read: the server startup commands already in README).

**Comment by justabluemouse** [comment]
Wow, that sounds like it would make updates a lot easier(and of course we would need a Stress Test 4 to test the new servers built on Heroku), so I'm all for it.

**Comment by luphoria** [comment]
Nonono, not the official servers. I just mean an option to deploy to Heroku for standalone users.

**Comment by AgentXLP** [comment]
Could work as a setup for private servers possibly?

**Comment by Bloxxel64** [comment]
this could actually work! The only problem is, we would need to switch to a node.js server.

**Comment by AgentXLP** [comment]
Not required no

**Comment by luphoria** [comment]
> this could actually work! The only problem is, we would need to switch to a node.js server.

No, there's no need for that - heroku supports rust no problem.
I suppose this issue would be a bit more useful in the rs repository, but i don't think that's much an issue :P

**Comment by AgentXLP** [comment]
holky shit luphoria sighting

**Comment by AgentXLP** [comment]
Also, I'm sure you're delighted at the "13 year old project manager"'s presence by the way, whether or not you miss the old gang is questionable at best actually.

Development seems to be still happening, although I wouldn't expect someone who hasn't worked on the game since approximately February 14th, 2021 to be fully in the loop.

**Comment by uuphoria2** [comment]
![image](https://github.com/sm64js/sm64js/assets/155935644/b14b7aec-2aaa-49f3-817c-259431142146)

**Comment by luphoria** [comment]
lol

**Comment by uuphoria2** [comment]
> lol

funny :trollface:

**Comment by queenkoopa8500** [comment]
luphoria replying after agent leaves sm64js lol
